client/landing/gutenboarding/onboarding-block/style-preview/font-select.tsx

```diff
@@ -42,7 +42,7 @@ const FontSelect: React.FunctionComponent = () => {
 		if ( ! selectedDesignDefaultFonts ) {
 			return true;
 		}
-		const [ defaultHeadings, defaultBase ] = selectedDesignDefaultFonts;
+		const { headings: defaultHeadings, base: defaultBase } = selectedDesignDefaultFonts;
```

Comment:
Now you can do `isShallowEqual(selectedDesignDefaultFonts, pair)`.

Reply:
Good suggestions. Do you mind applying them and merging?

Reply:
Not at all. On it.